lib/alerts/components/AlertEditor.js

```diff
@@ -44,9 +43,17 @@ export default class AlertEditor extends Component {
     this.props.onComponentMount(this.props)
   }
 
+  _stringToOption = str => (
```

Comment:
Please make this function live outside the class since it doesn't depend on any instance variables.